README.md

```diff
@@ -219,15 +219,16 @@ See [examples](https://github.com/ClickHouse/clickhouse-rs/tree/main/examples).
 * `lz4` (enabled by default) — enables `Compression::Lz4` and `Compression::Lz4Hc(_)` variants. If enabled, `Compression::Lz4` is used by default for all queries except for `WATCH`.
 * `native-tls` — supports urls with the `HTTPS` schema via `hyper-tls`, which links against OpenSSL.
 * `rustls-tls` — supports urls with the `HTTPS` schema via `hyper-rustls`, which does not link against OpenSSL.
+* `rustls-tls-aws` - use [`aws-lc-rs`](https://github.com/rustls/hyper-rustls?tab=readme-ov-file#crate-features) instead of the default pure Rust `ring` backend for `hyper-rustls`.
```

Comment:
Let's note that it requires a C compiler.

Reply:
latest aws-lc-rs build requirements are much improved